masterlist.yaml

```diff
@@ -1804,6 +1804,24 @@ plugins:
         subs: [ '**KFE - Refill Water Bottles.esp**' ]
         condition: 'checksum("KFE - Refill Water Bottles.esp", B85EF60C)'
 
+  - name: 'ROOGNV.esp'
+    url:
+      - link: 'https://www.nexusmods.com/newvegas/mods/76452/'
+        name: 'ROOG FNV'
+    req:
+      - *JIPLN
+      - 'YUP - Base Game + All DLC.esm'
+      - 'WeaponRepairKitRepaired.esp'
+      - 'WMIMNV.esp'
+    tag:
+      - C.Encounter
+      - Names
+      - Scripts
```

Comment:
Is the `Scripts` Bash Tag needed here? According to WBs [Advanced Readme](https://wrye-bash.github.io/docs/Wrye%20Bash%20Advanced%20Readme.html#ScriptsFO3) it refers to changes to the (SCRI) subrecord..